src/CEOFESABundle/Controller/StructureController.php

```diff
@@ -239,4 +241,28 @@ private function createDeleteForm($id)
             ->getForm()
         ;
     }
+
+    /**
+     * @param $id
+     *
+     * @Route("/editpdf/{id}", name="structure_pdf")
+     * @return Response
+     */
+    public function createPdf($id)
+    {
+        $em = $this->getDoctrine()->getManager();
```

Comment:
You don't really need to store the manager as you only use it once, can make sense for clarity but hey :)